modules/client/src/main/java/org/apache/ignite/internal/client/table/MapFunction.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.client.table;
+
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+import org.apache.ignite.internal.lang.IgniteTriFunction;
+
+/**
+ * Defines alias for batch map function.
+ *
+ * @param <K> Key type.
+ * @param <R> Result type.
+ */
+@FunctionalInterface
+interface MapFunction<K, R> extends IgniteTriFunction<Collection<K>, PartitionAwarenessProvider, Boolean, CompletableFuture<R>> {
```

Comment:
It is difficult to understand the purpose of every function parameter.

Let's remove `extends IgniteTriFunction` and add the function to the interface with explanatory parameter names and javadoc?

Reply:
🆗